wdl-lint/src/rules/section_order.rs

```diff
@@ -50,6 +52,15 @@ fn task_section_order(span: Span, name: &str, problem_span: Span) -> Diagnostic
         )
 }
 
+/// Create a sturct section order diagnostic.
+fn struct_section_order(span: Span, name: &str, problem_span: Span) -> Diagnostic {
+    Diagnostic::note(format!("sections are not in order for struct `{name}`"))
+        .with_rule(ID)
+        .with_label("this struct contains sections that are out of order", span)
+        .with_label("this section is out of order", problem_span)
+        .with_fix("order as `meta`, `parameter_meta`, `members`")
```

Comment:
```suggestion
        .with_fix("order as `meta`, `parameter_meta`, members")
```

In line with "private declarations" for `task` and `workflow`, I don't think this should be backticked as it isn't a formal section, but rather a collection of declarations.